sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -967,7 +967,7 @@ Scheduler::GraphBuildResult Scheduler::GraphBuilder::addCG(
           continue;
         }
         // Handle event dependencies on any commands part of another active
-        // fusion.
+        // fusion by aborting it.
```

Comment:
I would prefer the comment to include explicitly that this will cancel the fusion on a different queue where the current command has a dependency, rather than the current queue.

Reply:
I think this is more explicit now